CodeForBaltimore/Bmore-Responsive
# Carorm/issue 96

# Description

Added in the JWT login process and made it so that the user routes test requires the JWT as well. 

## Related PRs

List related PRs against other branches:

| branch          | PR       |
| --------------- | -------- |
| other_pr_description | # |

## Todos


## Deploy Notes

Notes regarding deployment the contained body of work. 

## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.

```sh

1. Load up the project (standard process and seed the DB)
2. From the root folder location - run NPM Test
3. See that the Auth Tests are run with the token

```

## Impacted Areas in Application

List general components of the application that this PR will affect:

- User routes tests
- Added utils/login.js
